### PR TITLE
fix: Translation Downloads deletion fix(celery)

### LINKED ISSUE #5766: Translations zip file deletion(Celery task) not working
**Describe the bug**
Translations donwloads as a zip file are generated properly but the deletion of the file as a celery task fails due to error given in the screenshot.

Reference: https://github.com/fossasia/open-event-frontend/pull/2445
55771111-47b4ed80-5ab9-11e9-8fbe-d24649d19bc5

**To Reproduce**
1. Go to 'Admin' Section
2. Click on 'Content tab'
3. Click on 'Download .po files'

**Expected behavior**
The deletion needs to happen properly. Adding to this, if we remove/comment the delete_translations code, the download works smoothly but the file must be removed as it shouldn't reside on the server at all.

**Additional context**
Working on it.

### PR DESCRIPTION
**Pull request by mrsaicharan1**
Fixes #5766

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
This resolves the `NoneType` assignment error arising because `celery.result.AsyncResult` returned by `apply_async` result is not being stored anywhere. 
 
#### Changes proposed in this pull request:

- Stored apply_async's return value in task

### CODE REVIEW

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 @iamareebjamal This PR is the last step involved in fixing the whole module of Translations Downloads. Please review this.

File: app/api/admin_translations.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def download_translations():
     shutil.move(zip_file_ext, temp_dir)
     path_to_zip = os.path.join(temp_dir, zip_file_ext)
     from .helpers.tasks import delete_translations
-    delete_translations.apply_async(path_to_zip, countdown=600)
+    task = delete_translations.apply_async(kwargs={'zip_file_path': path_to_zip}, countdown=600)
```
[the remark this answers is not in the document]
Comment (iamareebjamal) [reply app/api/admin_translations.py]: This doesn't make any sense

[the remark this answers is not in the document]
Comment (mrsaicharan1) [reply app/api/admin_translations.py]: @iamareebjamal My bad. I understood this wrong. Previously I was doing`delete_translations.apply_async(path_to_zip, countdown=600)` but that's not how the parameters are usually passed. We either pass in a dict of kwargs or a set of arguments with parenthesis. 
https://docs.celeryproject.org/en/latest/userguide/calling.html

**Comment by mrsaicharan1** [comment]
@CosmicCoder96 Please merge.

**Pull request merged**